### PR TITLE
Add 6.7.0 release highlights

### PR DESCRIPTION
**Pull request by danielmitterdorfer**
With this commit we document the highlight features of the Elasticsearch
6.7.0 release.

### CODE REVIEW

File: docs/reference/release-notes/highlights-6.7.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<titleabbrev>6.7.0</titleabbrev>
+++++
+
+See also <<release-notes-6.7.0,{es} 6.7.0 release notes>>.
+
+[float]
+=== Cross-cluster replication
+
+We have introduced cross-cluster replication in 6.5.0 as a beta feature and it
+is now mature enough that we make it generally available. Cross-cluster
```
Comment (zuketo) [inline docs/reference/release-notes/highlights-6.7.0.asciidoc]: We introduced cross-cluster replication in 6.5.0 as a beta feature, it
is now mature enough to declare the feature GA (general availability).

File: docs/reference/release-notes/highlights-6.7.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+for replicating data to a second cluster for disaster recovery purposes and for
+geo-proximity so that reads can be served locally.
+
+For more information, see {stack-ov}/xpack-ccr.html[Cross-cluster replication]
+and <<ccr-apis>>.
+
+=== Index lifecycle management
+
+We have introduced index lifecycle management in 6.6.0 as a beta feature and it
+is now mature enough that we make it generally available. The index lifecycle
```
Comment (zuketo) [inline docs/reference/release-notes/highlights-6.7.0.asciidoc]: Maybe the same wording from CCR: "We introduced index lifecycle management in 6.6.0 as a beta feature, it
is now mature enough to declare the feature GA (general availability).

File: docs/reference/release-notes/highlights-6.7.0.asciidoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* Move the old index to warm nodes, where it can be shrunk to a single shard and force-merged down to a single segment for optimized storage and querying. 
+* Later, move the index to cold nodes for cheaper storage.
+
+See <<index-lifecycle-management>>. 
+
+[float]
+=== {es-sql}
+
+We have introduced {es-sql} in 6.3.0 as an experimental feature and it is now
+mature enough that we make it generally available. This feature enables
```
Comment (zuketo) [inline docs/reference/release-notes/highlights-6.7.0.asciidoc]: We introduced {es-sql} in 6.3.0 as an experimental feature, it
is now mature enough to declare the feature GA (general availability).

**Review by zuketo — COMMENTED** [review]
A few minor suggestions, otherwise, lgtm

**Comment by danielmitterdorfer** [comment]
Thanks for the review. I've addressed all of your comments now in https://github.com/elastic/elasticsearch/pull/39826/commits/ee86b4f906c1d96ca67ac03561d8715558e91304.

**Review by zuketo — APPROVED** [review]
lgtm

**Comment by danielmitterdorfer** [comment]
@elasticmachine run elasticsearch-ci/docs-check

**Pull request merged**